webapp/channels/src/components/admin_console/revoke_token_button/revoke_token_button.tsx

```diff
@@ -16,30 +16,42 @@ interface RevokeTokenButtonProps {
     onError: (errorMessage: string) => void;
 }
 
-export default class RevokeTokenButton extends React.PureComponent<RevokeTokenButtonProps> {
-    private handleClick = async (e: React.MouseEvent) => {
+const RevokeTokenButton: React.FC<RevokeTokenButtonProps> = (props) => {
```

Comment:
There is no need to be explicit about the type. This can be rewritten as:
```suggestion
const RevokeTokenButton = (props: RevokeTokenButtonProps) => {
```